DEVELOPMENT.md

```diff
@@ -7,6 +7,31 @@
 * [Rustup](https://github.com/rust-lang/rustup)
 * Dependencies of [Cypress](https://docs.cypress.io/guides/getting-started/installing-cypress#System-requirements)
 
+## Running Upstream from source
+
+To start Upstream run `yarn start`.
+
+Running upstream with `yarn start` will use `<repo_root>/sandbox/rad_home` as
+the default `RAD_HOME` value to isolate you development state. To reset the
+application state remove `<repo_root>/sandbox/rad_home` and restart the
```

Comment:
Shouldn't we first stop the app, then remove the folder and then start again? 🤔

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2174/commits/ab01f8a2f9fbac3a582fedb10a6e87b019b6155e